3b_parameters/MODIS_MCD12Q1_V6/1_download/download_modis_mcd12q1_v6.ipynb

```diff
@@ -24,7 +24,9 @@
     "from netrc import netrc\n",
     "from pathlib import Path\n",
     "from shutil import copyfile\n",
-    "from datetime import datetime"
+    "from datetime import datetime\n",
+    "import concurrent\n",
+    "from concurrent.futures.thread import ThreadPoolExecutor"
```

Comment:
Is the line `import concurrent` needed?

Reply:
Yes, it is. I had an issue before doing the import this way.